frontend/src/pages/associations/elections/ActiveList.tsx

```diff
@@ -0,0 +1,241 @@
+import {PageTitle} from "../../../utils/common";
+import React, {useState} from "react";
+import {Link} from "react-router-dom";
+import {Button, Col, Modal} from "react-bootstrap";
+import Card from "react-bootstrap/Card";
+import Row from "react-bootstrap/Row";
+import {electionActiveStatus, api, useBetterQuery} from "../../../services/apiService";
+import {LoadingAssociation} from "../Loading";
+import {Election, Ballot, Choice} from "../../../models/associations/election";
+import {ChoiceButton, ListOfVotersButtonModal, toFrenchDate, toFrenchTimeNoSecondString} from "./Commons";
+
+export const AssociationElectionActiveList = ({ association }) => {
+
+    const associationId = association.id;
+    const { data, status, error } = useBetterQuery<Election[]>(
+        "elections.list",
+        api.elections.list,
+        associationId,
+        electionActiveStatus.Current
+    );
+
+    if (status === "loading") return <LoadingAssociation />;
+    else if (status === "error") return `Something went wrong: ${error}`;
+    else if (data) {
+        return (
+            <>
+                <PageTitle className={"mt-6"}>Élections en cours</PageTitle>
+                <Row>
+                    {data.map(election => {
+                        return (
+                            <Col md={5} key={election.id}>
+                                <ElectionCard election={election} association={association}/>
+                            </Col>
+                        );
+                    })}
+                </Row>
+            </>
+        )
+    }
+};
+
+const ElectionCard = ({ election, association }) => {
+    let editButton;
+    if (association.myRole.electionPermission) {
+        editButton = (
+            <Link
+                to={`/associations/${association.id}/elections/${election.id}/modifier`}
+                className={"float-right mt-0"}
+            >
+                <i className={"fe fe-edit"}/>
+            </Link>
+        );
+    } else {
+        editButton = (<></>)
+    }
+    const startsAt = new Date(election.startsAt);
+    const endsAt = new Date(election.endsAt);
```

Comment:
Why using an intermediary variable here?

Reply:
Because election.startsAt is a `string` I have to change to a `Date`